src/main/java/no/bibsys/Application.java

```diff
@@ -57,27 +62,32 @@ public static void main(String args[]) throws IOException {
         String action = System.getProperty("action");
         StringBuilder message = new StringBuilder(100);
         message.append("System property \"action\" is not set\n")
-            .append("Valid values: create,delete,update-role");
+            .append("Valid values: create,delete");
         Preconditions.checkNotNull(action, message.toString());
 
-        Application.run(repoOwner, repository, branch, action);
+        String swaggerOrg=System.getProperty("swaggerOrg");
```

Comment:
Do you have a environment reader helper class?

Reply:
I have, but these are system properties, i.e. java -DsystemProperty=systemValue